auction/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "orml-auction"
+version = "0.0.1"
+authors = ["Acala Developers"]
+edition = "2018"
+
+[dependencies]
+serde = { version = "1.0", optional = true }
+codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false }
+sr-primitives = { git = "https://github.com/paritytech/substrate.git", default-features = false }
+srml-support = { package = "srml-support", git = "https://github.com/paritytech/substrate.git", default-features = false }
+srml-system = { package = "srml-system", git = "https://github.com/paritytech/substrate.git", default-features = false }
+runtime_io = { package = "sr-io", git = "https://github.com/paritytech/substrate.git", default-features = false }
+rstd = { package = "sr-std", git = "https://github.com/paritytech/substrate.git", default-features = false }
+
+traits = { package = "orml-traits", path = "../traits", default-features = false }
+utilities = { package = "orml-utilities", path = "../utilities", default-features = false }
+
+[dev-dependencies]
+primitives = { package = "substrate-primitives",  git = "https://github.com/paritytech/substrate.git", default-features = false }
+
+[features]
+default = ["std"]
+std = [
+	"serde",
+	"codec/std",
+	"sr-primitives/std",
+	"srml-support/std",
+	"srml-system/std",
+	"runtime_io/std",
+	"rstd/std",
+	"traits/std",
+	"utilities/std",
+]
```

Comment:
It is recommended to install [EditorConfig](https://editorconfig.org/) in your IDE, to apply `insert_final_newline=true` and other rules automatically while editing.